### DISCUSSION TITLE
v2 Cannot get the focus inside the window

### DISCUSSION THREAD

**Discussion by yk0n9** [tauri-apps/tauri#12013] [Q&A]
I used the following function to make the tauri window top and get the focus
```rust
pub fn set_top_window(hwnd: HWND) {
    unsafe {
        let h_fore_wnd = GetForegroundWindow();
        let dw_fore_id = GetWindowThreadProcessId(h_fore_wnd, None);
        let dw_cur_id = GetCurrentThreadId();
        AttachThreadInput(dw_cur_id, dw_fore_id, TRUE);
        ShowWindow(hwnd, SW_SHOWNORMAL);
        SetWindowPos(hwnd, HWND_TOPMOST, 0, 0, 0, 0, SWP_NOSIZE | SWP_NOMOVE);
        SetWindowPos(hwnd, HWND_NOTOPMOST, 0, 0, 0, 0, SWP_NOSIZE | SWP_NOMOVE);
        SetForegroundWindow(hwnd);
        AttachThreadInput(dw_cur_id, dw_fore_id, FALSE);
    }
}
```
It is feasible in V1. After switching, the window will receive keyboard events.
But in V2, only the focus of the title bar is switched. Although the title bar is fixed, no keyboard events can be received inside the window, and the mouse must be used to actively obtain the focus.

Do I need to enable any permissions? Or other

**Comment by yk0n9** [comment]
remove unstable features

**Comment by iamxiaojianzheng** [comment]
Hello, may I ask if this issue has been resolved? 

I am currently encountering a problem where the input inside a window cannot gain focus automatically after the window is opened.
